.github/dependabot.yml

```diff
@@ -8,6 +8,24 @@ updates:
     labels:
       - "Maintenance"
       - "Dependencies"
+    ignore:
+      # Exclude main project dependencies and optional dependencies (like reader)
+      - dependency-name: "ansys-api-fluent"
```

Comment:
I am just curious that why these were not part of dependabot PRs earlier even though it was not part of ignore list. Specifically the ansys-* ones.

@mkundu1, any thoughts on this?